README.md

```diff
@@ -80,29 +79,21 @@ $ git clone git@github.com:architect-templates/react.git
 $ cd ./react
 
 # Deploy locally using the dev command
-$ architect dev .
+$ architect dev architect.yml
 ```
 ## Deploying to the Cloud
 
 Want to try deploying this to a cloud environment? Architect's got you covered there, too! It only takes a minute to
-[sign up for a free account](https://auth.architect.io/u/signup?state=hKFo2SAtSnhOdXljdy1nelBHb2NlajNhZkkybTlLOEJHcWRFeaFur3VuaXZlcnNhbC1sb2dpbqN0aWTZIFNCNEZUUFBHaWpBdlA3UVlVV0xFNk1rQVJvUHBzdF9Bo2NpZNkgbElwVzlmcTlJRlFCQmpUZ2xsaE42RUkwMVRYTWhSVm0).
+[sign up for a free account](https://cloud.architect.io/signup).
 
-You can then [register each component](https://docs.architect.io/getting-started/?_ga=2.19805311.635236263.1652126693-1328677302.1650395826#register-a-component)
-to your free environment and [deploy the application](https://docs.architect.io/getting-started/introduction/#deploy-to-the-cloud).
-
-To do this, run the following commands.
-The `<account-name>` is the name you used when you created your account in Architect Cloud. Use "example-environment" for
-the `<environment-name>` to deploy to the free environment that is created when you register with Architect.
+You can then [deploy the application](https://docs.architect.io/getting-started/introduction/#deploy-to-the-cloud). To do this, run the following commands. "example-environment" is the name of the free environment that is created when you register with Architect.
```

Comment:
Change to (keep the hyperlinks):
You can then deploy the application by running the command below. Note that “example-environment” is the free environment that is created with your Architect account.